Macros/Compile_ACS.sas

```diff
@@ -49,6 +50,27 @@
     %let geo_suffix = tr00;
     %let geo_var = Geo2000;
   %end;
+  %else %if &geo = CITY %then %do;
+    %** city level - use only for DC **;
+    %let _acs_sf_raw_path = &_acs_sf_raw_base_path\&_state_name._All_Geographies_Not_Tracts_Block_Groups;
+    %let sum_level = 040;
+    %let geo_suffix = city;
+    %let geo_var = city;
+  %end;
+  %else %if &geo = COUNTY %then %do;
```

Comment:
We could potentially account for both boundary and FIPS code changes, but it depends on the source data. 
We could leave these as they are now and then if at some point in the future we decide we need new versions, we can append the years to those names. I doubt we would want to do that before the next Census, in any case.